pikelet/src/lang/core/typing.rs

```diff
@@ -118,6 +120,9 @@ impl<'me> State<'me> {
     }
 
     /// Check that a term is an element of a type.
+    #[allow(unused_braces)] // FIXME: Bug in `contracts`?
```

Comment:
@karroffel This seems like a bug in `contracts`? Seems to only pop up in single expression function bodies, where there are no other imports/items nested inside.

Reply:
Oh, it seems like a newish lint?

Reply:
That should be fixed in the latest git version: https://gitlab.com/karroffel/contracts/-/commit/68c9e33274898524bf490a902cea8fb0cdedf9ad

Reply:
Yay! Thanks!